cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,19 @@
+const { generateUser } = require('../../../cypress/support/generate');
+
 describe('Sign Up page', () => {
+  before(() => {
+    cy.visit('/');
+  });
+
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
 
+    cy.get('[href="#register"]').click();
```

Comment:
good that you know this method, but for the future – it is not THE BEST practice. Try to use this when you really have no choice)